packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -7,21 +7,28 @@ propComponents:
     'Table',
     'TableHeader',
     'TableBody',
-    'EditableSelectInputCell',
-    'EditableTextCell',
     'TableComposable',
     'Thead',
     'Tbody',
     'Tr',
     'Th',
     'Td',
     'Caption',
+    'TableText',
+    'EditableSelectInputCell',
+    'EditableTextCell'
```

Comment:
add `EditableSelectInputProps`, `EditableTextCellProps`, `TdActionsType`, `TdSelectType`, `TdTreeRowType`,  `IActions`, `TdCompoundExpandType`, `TdFavoritesType`, `TdDraggableType`, `ThInfoType`,  and `TdExpandType` here.